### PR TITLE
Release 6.2.8

### PR DESCRIPTION
**Pull request by jtuttle**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [6.2.8] - 2022-08-16
+
+- Fixed rubygems delivery [cyberark/cyberark-conjur-cli-docker-based#354](https://github.com/cyberark/cyberark-conjur-cli-docker-based/pull/354)
```
Comment (gl-johnson) [inline CHANGELOG.md:11]:
```suggestion
### Fixed
- Fixed rubygems delivery [cyberark/cyberark-conjur-cli-docker-based#354](https://github.com/cyberark/cyberark-conjur-cli-docker-based/pull/354)
```

Comment (jtuttle) [reply CHANGELOG.md:12]: Doh, thanks. Fixed.

**Review by gl-johnson — DISMISSED** [review]
LGTM!

**Review by szh — APPROVED** [review]
LGTM!

**Pull request merged**